### ISSUE TITLE
The custom dropdown menu is not accessible to screen readers

### ISSUE THREAD

**Issue by luontola** [CSCfi/rems#1433]
NVDA+Firefox doesn't read the options, but only says "blank" for all of them.

**Comment by luontola** [comment]
Fixed in #1444
